src/parachain/amm.ts

```diff
@@ -111,13 +111,14 @@ export interface AMMAPI {
      * @param {MonetaryAmount<CurrencyExt>} minimumAmountOut Minimum output amount to be received.
      * @param {AddressOrPair} recipient Recipient address.
      * @param {number | string} deadline Deadline block for the swap transaction.
+     * @returns {ExtrinsicData} A submittable extrinsic and event.
```

Comment:
> A submittable extrinsic and event.

I wonder if this could be a bit confusing?
It returns a submittable extrinsic and the event we expect the parachain to emit when it is submitted / included. I wonder if we should be more explicit about that.